### PR TITLE
Rails-blog-app: Forms

### PR DESCRIPTION
**Pull request by Lichapa**
## Rails-blog-app: Forms

In this milestone, I have done the following

- Create forms to perform the following functions:
    - Creates a Post on behalf of the current_user.
    - Create a comment on behalf of the current_user.

- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    @comment = Comment.new(comment_params)
+    @comment.author_id = current_user.id
+    @comment.post_id = params[:post_id]
+
+    if @comment.save
+      redirect_to user_post_path(current_user.id, Post.find(params[:post_id]))
+    else
+      render :new
+    end
+  end
```
Comment (FrederickMih) [inline app/controllers/comments_controller.rb:18]: - [x] Nice job for including the `new `and `create` actions that can enable the current login user to add a comment for a post unfortunately, there's no form in views that accepts inputs for creating comments. Please can we consider adding a form for this aspect?. Also, remember the requirements demand us to have a form for adding comments.

File: app/controllers/likes_controller.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    @like = Like.new(likes_params)
+    @like.author_id = current_user.id
+    @like.post_id = params[:post_id]
+
+    if @like.save
+      redirect_to user_post_path(current_user.id, Post.find(params[:post_id]))
+    else
+      render new
+    end
+  end
```
Comment (FrederickMih) [inline app/controllers/likes_controller.rb:18]: - [x] I'm unable to add a like to a post. Please also make sure that a user should be able to add a like to a given post.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @Lichapa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



# To Highlighs

- Github workflow has been followed :+1: 
- Linters are all passing :heavy_check_mark: 
- I'm able to add a post :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (Lichapa) [reply app/controllers/comments_controller.rb:18]: Thanks @FrederickMih for the review. I have included the comment form in `posts/:id/show` to make a comment on the  post and Below is the link demonstrating this feature.

[Demo video ](https://www.loom.com/share/9271109c323549c4b42d6a90f12a9f52)

File: app/controllers/likes_controller.rb (hunk 2 above)
Comment (Lichapa) [reply app/controllers/likes_controller.rb:18]: I have added a link above demonstrating this feature

**Review by Lordkaito — APPROVED** [review]
Hi @Lichapa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Lichapa** [comment]
Thanks @Lordkaito

**Pull request merged**